src/util/message/message.h

```diff
@@ -30,18 +30,18 @@ class messaget
 
   /**
    * @brief Prints an error message
-   * 
+   *
    * @param message string to be printed
    * @param location location where the message happened
    */
   virtual void
   error(const std::string &message, const std::string location = "") const
   {
-    print(VerbosityLevel::Error, message, location);
+    print(VerbosityLevel::Error, "###ERROR: " + message, location);
```

Comment:
Thats nice, but it could be in a separate PR

Reply:
Agreed. I'll split this PR into multiple small PRs.